src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderRow.tsx

```diff
@@ -49,10 +56,9 @@ const QueryBuilderRow: React.FC<IQueryBuilderRowProps> = (props) => {
         onUpdateSnippet
     } = props;
     const propertySelectorId = useId('cb-advanced-search-property-select');
-    // Naming this as its type since theme is a prop as well
-    const iTheme = useTheme();
+    const theme = useTheme();
```

Comment:
`theme` isn't used elsewhere; I think you can skip the const. I used the `iTheme` const because theme was duplicated by the prop that base component needed.

Reply:
but you got rid of base compoent. So I figured we could put it back to the normal name.

Reply:
I was suggesting to remove the const and just call useTheme inside the getClassNames function.

Reply:
It's actually used further down for hte react select styles so I have to leave it as is. But I'll bring the styles next to eachother so it's easier to see that.